app/controllers/admin/welcome_controller.rb

```diff
@@ -1,16 +1,15 @@
 class Admin::WelcomeController < Admin::ApplicationController
   def index
-    batches = Batch.preload(:roast).order("starts_at asc").to_a
-    votes = Vote.all.to_a
+    batches = Batch.preload(:roast).order("starts_at desc")
+    report = CreateVotingReport.new(batches, Vote.all).perform
 
-    report = CreateVotingReport.new(batches, votes).perform
+    @active_batch = batches.active
 
-    @active_batch = Batch.active_batch(batches)
-    @active_roast = @active_batch.roast
-
-    @active_roast_scores = {:good => report[@active_roast][:good], :bad => report[@active_roast][:bad]}
-
-    @sorted_report = report.sort_by { |_, stats| stats[:score] }.reverse[0..9]
-    @top_10_batches = batches[0..9]
+    if !@active_batch.blank?
```

Comment:
I think you can use ```ruby if @active_batch.present?```
It is good practice to avoid `!` in conditions. Also prefer `if` to `unless` if possible